uhubctl.c

```diff
@@ -1029,6 +1033,9 @@ int main(int argc, char *argv[])
             continue;
         if (k == 1 && opt_action == POWER_KEEP)
             continue;
+        // if power_toggle is active: only one time
```

Comment:
Change comment to:

    // if toggle requested, do it only once when `k == 0`

Reply:
Ok